packages/usuario/src/useCases/usuario.service.js

```diff
@@ -0,0 +1,58 @@
+const boom = require('@hapi/boom');
+
+let usuarios = [
+	{ id: 1, persona_id: 1, rol_id: 2, usuario: 'admin', estatus: 1 },
+	{ id: 2, persona_id: 2, rol_id: 2, usuario: 'admin2', estatus: 1 },
+];
+
+async function getUsuarioById(data) {
+
+	const id = data;
+	// model user
```

Comment:
If we are not going to use this comments we should delete it, if we are going to keep let's make them clearer